src/OrchardCore.Modules/OrchardCore.Navigation/PagerShapes.cs

```diff
@@ -194,11 +194,11 @@ bool ShowNext
             }
 
 
-            var firstText = FirstText ?? T["<<"];
-            var previousText = PreviousText ?? T["<"];
-            var nextText = NextText ?? T[">"];
-            var lastText = LastText ?? T[">>"];
-            var gapText = GapText ?? T["..."];
+            var firstText = FirstText ?? "<<";
```

Comment:
Localized on purpose, to provide a way to customize the text without updating the shapes. Worst case, the localization is the same as the source, so no work needs to be done.

Reply:
If that's the intend here, I will undo the changes

Reply:
I rethink about this again, localize such thing will allow multiple values for different cultures

In case we need to change them we should allow that in views